src/client.h

```diff
@@ -272,6 +274,10 @@ class CClient : public QObject
         Channel.GetBufErrorRates ( vecErrRates, dLimit, dMaxUpLimit );
     }
 
+    // convert between MIDI index and real channel index
```

Comment:
Shouldn't this be done my the MIDI-handling code?  Otherwise, it could do with a longer comment explaining why it's here.